keyboards/hineybush/h75_singa/readme.md

```diff
@@ -0,0 +1,15 @@
+# h75_singa
+
+![h75_singa](imgur.com image replace me!) - TBD
```

Comment:
Need an image here :)

Reply:
The PCBs are in production, I plan to add an image once they "exist"